### PR TITLE
Fix language

### PR DESCRIPTION
**Pull request by viruxe**
Helps to determine in which language a key is missing.

### CODE REVIEW

**Comment by Southclaw** [comment]
Shall I copy some of these commits in manually? I don't think it's possible to pull specific commits from your fork.

**Comment by viruxe** [comment]
Oh sorry forgot about this. Just copy the commit related to the language extension. Ignore the rest.

File: scriptfiles-folder/SSS/languages/English [hunk 1]
```diff
@@ -64,3 +64,6 @@ REFUELLING=Refuelling...
 TENTREMOVE=Removing tent...
 NEEDWORKBE=You need a workbench to combine those
 NEARBYKEYP=This door is electronically locked controlled by a nearby keypad.
+HOLSTERHOLS=Holstered smth
+HOLSTEREQUI=Equipped Holster
+BAGITEMADDED=Item added to bag
```
Comment (Southclaw) [inline scriptfiles-folder/SSS/languages/English]: The holster keys should already be in the latest copy at line 75. The bag line also is but it has one less character at line 71

Comment (viruxe) [inline scriptfiles-folder/SSS/languages/English]: Yes you can ignore those. It was from when it was missing.

**Pull request closed without merging**